jvm/src/main/scala/io/glutenproject/execution/WholeStageTransformerExec.scala

```diff
@@ -320,7 +324,27 @@ case class WholeStageTransformerExec(child: SparkPlan)(val transformStageId: Int
         s"${execTempDir}/spark-columnar-plugin-codegen-precompile-${signature}.jar"
       })
       val fileScan = current_op.get
+      // Get the file format based on fileScan
+      val fileFormat = fileScan match {
+        case f: BatchScanExecTransformer => {
+          f.scan.getClass.getSimpleName match {
+            case "OrcScan" => 2
+            case "ParquetScan" => 1
+            case "DwrfScan" => 3
```

Comment:
Can current Velox handle above values? I can only see the handling for 0 and 1. 
```
      if (format == 0) {
        splitInfo->format = dwio::common::FileFormat::ORC;
      } else if (format == 1) {
        splitInfo->format = dwio::common::FileFormat::PARQUET;
      } else {
        splitInfo->format = dwio::common::FileFormat::UNKNOWN;
      }
```

Reply:
Update in [PR#21](https://github.com/oap-project/velox/pull/21)